Class/Practise 3.2/src/Triangle.cpp

```diff
@@ -0,0 +1,9 @@
+#include "Triangle.hpp"
+#include <cmath>
+
+Triangle::Triangle(double a, double b, double c) : a(a), b(b), c(c) {}
+
+double Triangle::area() const {
+    double s = (a + b + c) / 2;
+    return std::sqrt(s * (s - a) * (s - b) * (s - c));
```

Comment:
Something wrong with your TDD. What about the case when s-a is negative or some of the variables a, b, c are negative.
Also all the triangles you have in your unit tests are right triangles which looks like for all of them the simplest way to calculate is to `return a * b / 2;` and all your tests will be green